packages/medusa/src/repositories/money-amount.ts

```diff
@@ -57,8 +84,15 @@ export const MoneyAmountRepository = dataSource
       prices: Price[]
     ): Promise<MoneyAmount[]> {
       const pricesNotInPricesPayload = await this.createQueryBuilder()
-        .where({
+        .leftJoinAndSelect(
+          "product_variant_money_amount",
+          "mav",
```

Comment:
suggestion: pvma, I think mav is a relic from a previous table name right? can you change everywhere?